docs/configuration.md

```diff
@@ -70,6 +70,18 @@ E.g.
 $ UNISON_SHARE_ACCESS_TOKEN="my.token.string" ucm
 ```
 
+### `UNISON_DISABLE_FILE_WATCH`
+
+Disables file-watching behaviour if set to any value (even the empty string).
+
+Use the `.> load scratch.u` command to load unison code instead.
+
+E.g.
+
+```sh
+$ UNISON_DISABLE_FILE_WATCH= ucm
```

Comment:
```suggestion
$ UNISON_DISABLE_FILE_WATCH=ucm
```

Reply:
@ceedubs the original is correct 😛

Reply:
Ohhh haha I was thrown off by it being set to the empty string. I clearly didn't think about this very hard. Please ignore.